File: app/src/main/java/com/example/imagemachine/data/model/Machine.java
package com.example.imagemachine.data.model;

import android.arch.persistence.room.ColumnInfo;
import android.arch.persistence.room.Entity;
import android.arch.persistence.room.PrimaryKey;

@Entity(tableName = "machine")
public class Machine {

    @PrimaryKey
    @ColumnInfo(name = "id")
    private Long id;

    @ColumnInfo(name = "name")
    private String name;

    @ColumnInfo(name = "type")
    private String type;

    @ColumnInfo(name = "qrCode")
    private String qrCode;

    @ColumnInfo(name = "date")
    private String date;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getQrCode() {
        return qrCode;
    }

    public void setQrCode(String qrCode) {
        this.qrCode = qrCode;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }
}

File: app/src/main/java/com/example/imagemachine/data/model/Machine.kt
package com.example.imagemachine.data.model

import android.arch.persistence.room.ColumnInfo
import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey

@Entity(tableName = "machine")
data class Machine(@PrimaryKey @ColumnInfo(name = "id") var id: Long,
                   @ColumnInfo(name = "name") var name: String,
                   @ColumnInfo(name = "type") var type: String,
                   @ColumnInfo(name = "qrCode")var qrCode: String,
                   @ColumnInfo(name = "date")var date: String)